src/arrayReverse.test.js

```diff
@@ -3,18 +3,66 @@
 describe(`Function 'arrayReverse':`, () => {
   const arrayReverse = require('./arrayReverse');
 
-  it(`should be declared`, () => {
-    expect(arrayReverse).toBeInstanceOf(Function);
+  it(`should be declared`, () => expect(arrayReverse).toBeDefined());
+
+  it(`should return an array of strings`, () => {
+    const result = arrayReverse(['One', 'two']);
+
+    expect(result).toBeInstanceOf(Array);
+    result.forEach((word) => expect(typeof word).toBe('string'));
   });
 
-  it(`should return an array`, () => {
+  it(`should return an array of the same length`, () => {
+    const array = ['One', 'two', 'three'];
+
+    expect(arrayReverse(array).length).toBe(array.length);
+  });
 
+  it(`should return a reversed array containing one word`, () => {
+    expect(arrayReverse(['Hello'])).toEqual(['olleH']);
   });
 
-  it(`should return an empty string
-    if original array consists of an empty string`, () => {
+  it(`should return a reversed array containing multiple words`, () => {
     expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
   });
 
-  // write more tests here
+  it(`should return '['']' if original array has only empty string`, () => {
+    expect(arrayReverse([''])).toEqual(['']);
+  });
+
+  it(
+    `should return array of reversed strings consisting of words and numbers`,
+    () => {
+      expect(arrayReverse(['one1', 'two2', 'three3']))
+        .toEqual(['3eer', 'ht2o', 'wt1eno']);
```

Comment:
The expected result for reversing the array `['one1', 'two2', 'three3']` should be `['3eerht', '2owt', '1eno']` according to the task description. Please verify the expected output.